runtime/marshaler_registry.go

```diff
@@ -13,7 +13,7 @@ var (
 	acceptHeader      = http.CanonicalHeaderKey("Accept")
 	contentTypeHeader = http.CanonicalHeaderKey("Content-Type")
 
-	defaultMarshaler = &JSONPb{OrigName: true}
+	defaultMarshaler = &HTTPBodyMarshaler{}
```

Comment:
This seems incorrect. Why does the default marshaller need to change from this change?

Reply:
why shouldn't the default behavior be to treat HTTP body protos specially?

Reply:
It is a breaking change. I try not to make those except on major version bumps. If you want to be added to the list of 2.0 breaking changes and do it then, I'd be happy to do that.